### DISCUSSION TITLE
想要修改节点和边的边框样式

### DISCUSSION THREAD

**Discussion by cfc3058353263** [didi/LogicFlow#1254] [General]
使用版本 v1.2.10
使用以下方法无效 
 lf.on('node:click', ({ data }) => {
    const model = lf.graphModel.getElement(data.id)
    model.fill = 'red'
  })
使用该方法只能修改fill样式不能修改stroke样式
 let node = lf.getNodeModelById(item.key);
 node && node.setStyles({ stroke: '#000', fill: '#edd48ec9' });

**Comment by wumail** [comment]
1、model上没有fill属性，样式都放在了model.style里，设置model.fill不会生效
2、第二种方法亲测有效，根据你所提供的信息无法判断出哪有问题

**Comment by yuguaa** [comment]
bump不生效

**Reply by wumail** [reply]
bump是指？
